noir-projects/noir-protocol-circuits/crates/rollup-lib/src/components.nr

```diff
@@ -125,16 +124,17 @@ pub fn compute_txs_effects_hash(previous_rollup_data: [PreviousRollupData; 2]) -
 // 1 field for transaction fee
 // MAX_NEW_NOTE_HASHES_PER_TX fields for note hashes
 // MAX_NEW_NULLIFIERS_PER_TX fields for nullifiers
-// 2 l2 -> l1 messages -> 2 fields
+// MAX_NEW_L2_TO_L1_MSGS_PER_TX fields for L2 to L1 messages
 // MAX_TOTAL_PUBLIC_DATA_UPDATE_REQUESTS_PER_TX public data update requests -> MAX_TOTAL_PUBLIC_DATA_UPDATE_REQUESTS_PER_TX * 2 fields
-// 1 contract deployments -> 3 fields
```

Comment:
There are no contract deployments anymore which was really confusing because there was number "+3" in its place in the constant computation. Instead the +3 corresponded to 3 log length containing fields for 3 log types.